### PR TITLE
ORM::factory-Update-5Files

### PR DESCRIPTION
**Pull request by Enoc-GitHub**

### CODE REVIEW

File: public_html/application/classes/Controller/Admin.php [hunk 1]
```diff
@@ -739,10 +747,12 @@ public function giveitems()
 			//var_dump( $_POST ); exit; 
 			$post = Validation::factory($this->request->post());
 	
-			$par[0] = ORM::factory( 'character' ) 
-				-> where ( array( 'name' => $this->request->post('to_username' ) )) -> find(); 
-			$par[1] = ORM::factory('cfgitem') 
-				-> where( 'id', $this -> request -> post('item'))->find();	
+			$par[0]=ORM::factory('character')
```
Comment (lothropgauss) [inline public_html/application/classes/Controller/Admin.php:750]: Don't remove spaces around assignment.

File: public_html/application/classes/Controller/Admin.php [hunk 2]
```diff
@@ -839,8 +851,8 @@ public function viewwardroberequest( $id = null )
 		
 		if ( !$_POST )
 		{
-			$request = ORM::factory('wardrobe_approvalrequest', $id );	
-			$character = ORM::factory('character', $request -> character -> id);
+			$request=ORM::factory('wardrobe_approvalrequest', $id);
```
Comment (lothropgauss) [inline public_html/application/classes/Controller/Admin.php:854]: same here, the space at the end is correct, but keep the one around assignment.

File: public_html/application/classes/Controller/Affiliate.php [hunk 3]
```diff
@@ -326,7 +325,9 @@ public function _checkoldpassword (Validation $array, $field)
   public function _unique_username(Validation $array, $field)
   {
      // controllo il db
-     $name_exists = (bool) ORM::factory('user')->where('username', $array[$field])->count_all();
```
Comment (lothropgauss) [inline public_html/application/classes/Controller/Affiliate.php:329]: This needs an equals

File: public_html/application/classes/Controller/Affiliate.php [hunk 4]
```diff
@@ -345,7 +346,9 @@ public function _unique_username(Validation $array, $field)
   public function _unique_email(Validation $array, $field)
   {
      // controllo il db
-     $email_exists = (bool) ORM::factory('user')->where('email', $array[$field])->count_all();
```
Comment (lothropgauss) [inline public_html/application/classes/Controller/Affiliate.php:348]: same as above

File: public_html/application/classes/Controller/Barracks.php [hunk 5]
```diff
@@ -69,11 +69,9 @@ function manageprisoners( $structure_id )
 			HTTP::redirect('region/view/');
 		}			
 		
-		$prisoners = ORM::factory("character_sentence")
```
Comment (lothropgauss) [inline public_html/application/classes/Controller/Barracks.php:72]: This should be where->(...)->and_where(...)->and_where(...)

**Review by lothropgauss — CHANGES_REQUESTED** [review]
Some changes around formatting and a few around the where functions.

Also for formatting you have use the intellisense addon, and rightclick -> format file, for any file you end up working on and that will do the formatting automatically, it kills the spaces that I don't like :)

File: public_html/application/classes/Controller/Affiliate.php (hunk 3 above)
Comment (Enoc-GitHub) [reply public_html/application/classes/Controller/Affiliate.php:329]: Where so?

Comment (lothropgauss) [reply public_html/application/classes/Controller/Affiliate.php:329]: `where('username', $array[$field])`
should be
`where('username', '=', $array[$field])`

**Pull request closed without merging**